include/mxnet/c_api.h

```diff
@@ -313,6 +313,13 @@ MXNET_DLL int MXSetProcessProfilerState(int state, int profile_process,
  */
 MXNET_DLL int MXSetProfilerState(int state);
 
+/*!
+ * \brief Set the scope of profiler for current process
```

Comment:
What possible values can go in the scope?

Reply:
@ChaiBapchya As an example, for Gluon applications, the profiler scope is set to be the Gluon block name.